inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/two_fake_quantize_to_fullyconnected.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include <gna/gna_config.hpp>
+
+#include "subgraph_tests/two_fake_quantize_to_fullyconnected.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32, InferenceEngine::Precision::FP16
+};
+
+using ConfigType = std::map<std::string, std::string>;
+const ConfigType configFP32 = {
+        {"GNA_DEVICE_MODE", "GNA_SW_FP32"},
+};
+
+
+/**
+ * @brief specific quantisation mode to be used internally
+ */
+const std::vector<std::pair<std::string, ConfigType>> gnaQuantModes = {
+            {"sw_fp32", configFP32},
+    };
+
+const std::vector<std::vector<size_t>> inputShapes = {
+        {1, 440}
+};
+const std::vector<std::vector<size_t>> constShapes = {{1}};
+const std::vector<size_t> levels = {255, 65535};
+
+const std::vector<std::vector<float>> fqArgs = {{0, 10, 2, 6}};
+const std::vector<std::vector<float>> inputParams = {{-1000, 1000, 10.0}, {-10, 10, 0.1}, {}};
```

Comment:
For int8, there is very limited precision. The range of values that needs to be represented is way to wide.